.github/workflows/python_metapackages.yml

```diff
@@ -0,0 +1,220 @@
+            exit 1
+          fi
+
+          $python -m pip install cudaq==${{ inputs.cudaq_version }} \
+            --extra-index-url http://localhost:8080 
+
+          if [ -z "$($python -m pip list | grep $package_dep)" ]; then 
+            echo "::error file=python_metapackages.yml::Missing installation of $package_dep package."
+            exit 1
+          elif [ "$($python -c 'import cudaq; print(cudaq.get_target().name)')" != "qpp-cpu" ]; then 
+            echo "::error file=python_metapackages.yml::Unexpected output for cudaq.get_target()."
+            exit 1
+          fi
+
+          $autoremove -y cudaq
+          if [ -n "$($python -m pip list | grep $package_dep)" ]; then 
+            echo "::error file=python_metapackages.yml::Unexpected installation of $package_dep package."
+            exit 1
+          fi
+
+          if [ "$package_dep" != "cuda-quantum" ]; then
+            $python -m pip install ${package_dep}==${{ inputs.cudaq_version }} \
+              --extra-index-url http://localhost:8080 
+            $python -m pip install cudaq==${{ inputs.cudaq_version }} \
+              --extra-index-url http://localhost:8080 
+            $python -c 'import cudaq; print(cudaq.get_target().name)'
+            $autoremove -y cudaq
+            if [ -z "$($python -m pip list | grep $package_dep)" ]; then 
+              echo "::error file=python_metapackages.yml::Missing installation of $package_dep package."
+              exit 1
+            fi
+            $autoremove -y $package_dep
+          fi
+
+      - name: Test installation error
+        run: |
+          #dnf install -y --nobest --setopt=install_weak_deps=False curl jq
+          #PACKAGE_JSON_URL="https://pypi.org/pypi/cuda-quantum/json"
+          #curl -L -s "$PACKAGE_JSON_URL" | jq  -r '.releases | keys | .[]'
+
+          python=python${{ matrix.python_version }}
+          $python -m pip install pypiserver
+          server=`find / -name pypi-server -executable -type f`
+          $server run -p 8080 /tmp/packages & 
+
+          case "${{ matrix.cuda_version }}" in
+            "") 
+              set +e # continue on error
+              $python -m pip install cuda-quantum==${{ inputs.cudaq_version }} \
+                --extra-index-url http://localhost:8080 
+              set -e && check_package=cuda-quantum
+            ;; 
+            12.*) 
+              $python -m pip install cuda-quantum-cu11==${{ inputs.cudaq_version }} \
+                --extra-index-url http://localhost:8080 
+              set +e # continue on error
+              $python -m pip install cudaq==${{ inputs.cudaq_version }} \
+                --extra-index-url http://localhost:8080
+              set -e && check_package=cudaq
+            ;; 
+            *) 
+              set +e # continue on error
+              # test that we get an error for an existing cuda-quantum installation
+              $python -m pip install cuda-quantum==0.8.0
```

Comment:
We will never change this to a newer one, because we will stop releasing new versions for cuda-quantum. Going forward, the CUDA-Q wheel will be released under the name cudaq.